eng/Version.Details.xml

```diff
@@ -105,7 +105,7 @@
       <Uri>https://github.com/dotnet/cli-migrate</Uri>
       <Sha>37e3a01aa1df320a286b0a858876f9994fb6bb18</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.CommandLineUtils" Version="1.0.1">
+    <Dependency Name="Microsoft.Extensions.CommandLineUtils" Version="3.0.0-preview5.19227.9">
```

Comment:
Another case of Version changes, Sha doesn't.

Reply:
This was a result of aspnet/common moving to Arcade, which changes their version number algorithm.  No code changes.